mhw_library/measures.py

```diff
@@ -0,0 +1,553 @@
+import numpy as np
+import xarray as xr
+from skimage import img_as_float
+from skimage.morphology import convex_hull_image
+from skimage.measure import label as label_np, regionprops, find_contours
+from scipy import ndimage
+from scipy.interpolate import interp1d
+from haversine import haversine, Unit
+from typing import Tuple
+
+def calc_duration(
+    merged_xarray : xr.Dataset, 
+    mhw_id : int) -> int:
+    """
+    Calculates the number of timestamps the object identifier
+    appears in the merged_xarray
+    Parameters
```

Comment:
You should add empty lines before "Parameters" and "Returns." This is a helpful style guide for docstrings that I found: https://pandas.pydata.org/docs/development/contributing_docstring.html#